structs.wdl

```diff
@@ -22,4 +22,9 @@ struct SampleConfig {
     Array[Sample] samples
 }
 
+struct Hisat2Index {
+    File directory
+    String basename
```

Comment:
It continues to amaze me what a mess indexes are to implement in pipelines. Star does it quite well actually.